tests/unit/Rules/LengthTest.php

```diff
@@ -13,120 +13,57 @@
 
 namespace Respect\Validation\Rules;
 
-use PHPUnit\Framework\TestCase;
+use Respect\Validation\Test\RuleTestCase;
 
 /**
  * @group  rule
+ *
  * @covers \Respect\Validation\Exceptions\LengthException
```

Comment:
This test doesn't cover `LengthException`, that's why we have the integration tests :)